parity/rpc_apis.rs

```diff
@@ -106,6 +108,7 @@ impl FromStr for Api {
 			"signer" => Ok(Signer),
 			"traces" => Ok(Traces),
 			"web3" => Ok(Web3),
+			"transactions_pool" => Ok(TransactionsPool),
```

Comment:
Is this API specified anywhere? If not I'd prefer to keep it under `parity` prefix (`Parity` namespace should be fine)